### ISSUE TITLE
Cannot read property 'position' of undefined

### ISSUE THREAD

**Issue by robmcguinness** [nearform/node-clinic-bubbleprof#355]
Using `clinic@5.x` with `bubbleprof` command and Chrome `80`, I see this  in browser console when loading the web interface:

```
66296.clinic-bubbleprof.html:2216 Uncaught TypeError: Cannot read property 'position' of undefined
    at module.exports.setCoordinates (66296.clinic-bubbleprof.html:2216)
    at module.exports.initializeFromData (66296.clinic-bubbleprof.html:2216)
    at SVGGElement.<anonymous> (66296.clinic-bubbleprof.html:2216)
    at Selection.each (66296.clinic-bubbleprof.html:2216)
    at module.exports.initializeFromData (66296.clinic-bubbleprof.html:2216)
    at BubbleprofUI.<anonymous> (66296.clinic-bubbleprof.html:2216)
    at emitNone (66296.clinic-bubbleprof.html:2216)
    at BubbleprofUI.EventEmitter.emit (66296.clinic-bubbleprof.html:2216)
    at BubbleprofUI.setData (66296.clinic-bubbleprof.html:2216)
    at 66296.clinic-bubbleprof.html:2216
setCoordinates @ 66296.clinic-bubbleprof.html:2216
initializeFromData @ 66296.clinic-bubbleprof.html:2216
(anonymous) @ 66296.clinic-bubbleprof.html:2216
each @ 66296.clinic-bubbleprof.html:2216
initializeFromData @ 66296.clinic-bubbleprof.html:2216
(anonymous) @ 66296.clinic-bubbleprof.html:2216
emitNone @ 66296.clinic-bubbleprof.html:2216
EventEmitter.emit @ 66296.clinic-bubbleprof.html:2216
setData @ 66296.clinic-bubbleprof.html:2216
(anonymous) @ 66296.clinic-bubbleprof.html:2216
setTimeout (async)
(anonymous) @ 66296.clinic-bubbleprof.html:2216
require.34../data/index.js @ 66296.clinic-bubbleprof.html:2216
o @ 66296.clinic-bubbleprof.html:2048
r @ 66296.clinic-bubbleprof.html:2048
(anonymous) @ 66296.clinic-bubbleprof.html:2048
66296.clinic-bubbleprof.html:1 
```

```
Access to font at 'https://clinicjs.org/assets/fonts/archia-semibold.woff2' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

Downgrading to `clinic@4.2.0` resolves my issue.

**Comment by DylanC** [comment]
@robmcguinness - I actually came across this issue yesterday too. We will look into it soon!

**Comment by robmcguinness** [comment]
```
❯ clinic bubbleprof -v
v3.0.2
```

**Comment by goto-bus-stop** [comment]
Ahh I guess we missed a few `sourceLayoutNode` cases in visualizer/draw/svg-node.js. @DylanC can you try to submit a PR updating the remaining ones today? then I will do a quick patch release afterward

**Comment by mcollina** [comment]
I just stumbled into this as well!

**Comment by DylanC** [comment]
@goto-bus-stop - I have a linked PR ready to go.

**Comment by goto-bus-stop** [comment]
Released as :package: 3.0.3. Thanks!
